src/helpers/pythonToFrames.ts

```diff
@@ -720,16 +754,17 @@ function makeAndAddFrameWithBody(p: ParsedConcreteTree, frameType: string, keywo
     return {s: s, frame: frame};
 }
 
-// Process the given node in the tree at the current point designed by CopyState 
+// Process the given node in the tree at the current point designed by CopyState,
+// and maintain an object, extraInfos, for things that need to be deferred (for example, multilines comments).
 // Returns a copy state, including the frame ID of the next insertion point for any following statements
-function copyFramesFromPython(p: ParsedConcreteTree, s : CopyState) : CopyState {
+function copyFramesFromPython(p: ParsedConcreteTree, s : CopyState, extraInfos: {multiLinesCommentContent: string | null} = {multiLinesCommentContent: null}) : CopyState {
```

Comment:
I would put multiLinesCommentContent into CopyState, it's already doing this kind of role (keeping track of state across the parsing).